openage/cvar/cvar.pyx

```diff
@@ -0,0 +1,17 @@
+# Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+from libopenage.cvar.cvar cimport load_config as c_load_config, get_cvar_manager
+
+from .config_file import load_config_file as load_config
+
+cdef void load_config_file() except * with gil:
+    load_config("config.cfg")
+
+cpdef set_cvar(name, value) with gil:
+    get_cvar_manager().set(name.encode('UTF-8'),value.encode('UTF-8'))
+
+cpdef get_cvar(name) with gil:
+    get_cvar_manager().get(name.encode('UTF-8'))
+
+def setup():
+    c_load_config.bind0(load_config_file)
```

Comment:
the `c_load_config` is then a member of the cvarmanager and can still be bound that way.